superset/assets/src/visualizations/deckgl/DeckGLContainer.jsx

```diff
@@ -23,12 +24,13 @@ const defaultProps = {
 export default class DeckGLContainer extends React.Component {
   constructor(props) {
     super(props);
+    this.tick = this.tick.bind(this);
+    this.onViewportChange = this.onViewportChange.bind(this);
```

Comment:
nit: use autobind here
https://github.com/andreypopp/autobind-decorator

Reply:
We should introduce `autobind` all at once, would rather not have it only in some places.